grails-app/controllers/org/bbop/apollo/GroupController.groovy

```diff
@@ -74,14 +77,14 @@ class GroupController {
             // restricted groups
             def groups = dataObject.groupId ? [UserGroup.findById(dataObject.groupId)] : UserGroup.all
             def filteredGroups =  groups
+            def currentUser = User.findByUsername(dataObject.username)
```

Comment:
is this not always pemrissionService.currentUser?  

That would be preferable as you know that user is logged in as the user of interest, unless it ihas caused a problem.

Reply:
It works for UI login in user. But not for web services API call. With API call, there's no pemrissionService.currentUser.

Reply:
Okay.  This is fine then.     Thanks for catching this.